package util

import "log"

func HandleFunc(err error) {
	log.Printf("Error occurred: %v", err)
}
